<?xml version="1.0"?>
<launch>
  <!-- Argument to the launch file.-->
  <arg name="counter_delay_parameter" default="1.0"/>

  <!-- Start the metres_to_feet service server node.-->
  <node name="meters_to_feet" pkg="senspublisher" type="meters_to_feet_server.py"
    output="screen"/>

  <!-- Start the counter with delay action server node /-->
  <node name="counter_with_delay" pkg="senspublisher" type="counter_with_delay_as.py"
    output="screen">
    <param name="counter_delay" type="double" value="$(arg counter_delay_parameter)"/>
  </node>

</launch>
